lib/matplotlib/axes/_axes.py

```diff
@@ -3237,22 +3237,26 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
 
         Other Parameters
         ----------------
-        The following boolean options toggle the drawing of individual
-        components of the boxplots:
-            - showcaps: the caps on the ends of whiskers
-              (default is True)
-            - showbox: the central box (default is True)
-            - showfliers: the outliers beyond the caps (default is True)
-            - showmeans: the arithmetic means (default is False)
-
-        The remaining options can accept dictionaries that specify the
-        style of the individual artists:
-            - capprops
-            - boxprops
-            - whiskerprops
-            - flierprops
-            - medianprops
-            - meanprops
+        showcaps : bool, optional (True)
+            Show the caps on the ends of whiskers.
+        showbox : bool, optional (True)
+            Show the central box.
+        showfliers : bool, optional (True)
+            Show the outliers beyond the caps.
+        showmeans : bool, optional (False)
+            Show the arithmetic means.
```

Comment:
They're actually subtly different; on `show*`, the "Show the ..." is an imperative command to the `boxplot` function, whereas on `*prop`, the "Specifies ..." is a description of the parameter itself.

Of course, I'm just speculating because @jenshnielsen wrote it originally.